codonPython/datetime_utils.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+
+def add_time(date : str, days : int = 0, months : int = 0, years : int = 0) -> str: 
+    
+    """
+    This function takes a date as a string input in the 'yyyy-mm-dd' format and returns 
+    transpose of the date, given the specified number of days, months and years. 
+    To substract days, months, years from the date passed, use negative values. 
+    
+    Parameters
+    -----------
+    date : string 
+        Original date
+        
+    days : int
+        Days to add/substract
+        default = 0
+        
+    months : int
+        Months to add/substract
+        default = 0
+        
+    years : int
+        Years to add/substract
+        default = 0
+        
+    Returns
+    -------
+    str
+    
+    Examples
+    -------
+    >>>add_time('2018-03-04', months = 4)
+    '2018-07-04'
+     
+    >>>add_time('2018-03-04', days = -90)
+    '2017-12-04'
+
+    >>>add_time('2018-03-04', years = 2)
+    '2020-03-04'
+
+    """
+    
+    return str((datetime.datetime.strptime(date,'%Y-%m-%d') + relativedelta(days = int(days), months = int(months), years = int(years))).strftime(('%Y-%m-%d')))

```

Comment:
Hi @RomanPenza,

Please could you ensure that there are no spaces around "=" when indicating a key argument. This is in compliance with PEP 8 formatting.

For example,

`relativedelta(days=int(days), months=int(months), years=int(years))`

Reply:
Hi, 

I get an error:
"Plugin 'pytest_cov' could not be loaded: (pytest 4.3.1 (/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages)"

Not sure where it comes from...

Reply:
Hi @RomanPenza, Apologies. There is currently a bug within the master build regarding the docs deployment. We are aiming to fix it in the near future, but there are no given timescales at the moment as the majority of the team is busy. Once the bug has been resolved I will let you know and we can proceed.

Reply:
Hi @RomanPenza 

We have fixed the bug with docs deployment. However, we have also updated our [contribution guide](https://github.com/codonlibrary/codonPython/blob/master/CONTRIBUTING.md) and currently conflicts with bullet point 3:
`Duplicated functionality. For example, if your function is already done by another well known package.`

In my opinion, the functionality that you have wrote can already be accessed using the datetime package, which is why I have challenged your merge request. If you disagree, please let me know by the 20th November or I will have to reject the merge request.

Apologies about this, I did originally agree to accept the function, but all active and future merge requests must adhere to the updated contribution guide.